python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -472,6 +477,7 @@ class TickData(
             ("end_timestamp", Optional[float]),  # Time the tick finished
             ("run_requests", Optional[Sequence[RunRequest]]),  # run requests created by the tick
             ("auto_materialize_evaluation_id", Optional[int]),
+            ("reserved_run_ids", Optional[Sequence[str]]),
         ],
     )
 ):
```

Comment:
nit: docstring has gotten out of date. Would be helpful to clarify that reserved_run_ids map to run_requests in the same order